### PR TITLE
New CSS features

### PR DESCRIPTION
**Pull request by Antonio-Laguna**
This adds support for

* Exponential Functions (`pow()`, `sqrt()`, `hypot()`, `log()`, `exp()`):  https://www.w3.org/TR/css-values-4/#exponent-funcs
* `calc()` constants (`e`, `pi`, `infinity`, `-infinity` and `NaN`) https://www.w3.org/TR/css-values-4/#calc-constants
* Sign functions (`abs()` and `sign()`): https://www.w3.org/TR/css-values-4/#sign-funcs
* Trigonometric functions (`sin()`, `cos()`, `tan()`, `asin()`, `acos()`, `atan()` and `atan2()`): https://www.w3.org/TR/css-values-4/#trig-funcs

Needed for https://github.com/csstools/postcss-plugins/pull/356

It also allows to manually override support for things that aren't yet on MDN nor Can I Use

### CODE REVIEW

**Comment by Antonio-Laguna** [comment]
@romainmenke if you can think of better examples it's more than welcome. The specs are very vague as they stand!

File: cssdb.settings.json [hunk 1]
```diff
@@ -809,6 +818,15 @@
       }
     ]
   },
+  {
+    "id": "calc-constants",
+    "title": "`e`, `pi`, `infinity`, `-infinity and `NaN` constants for calculation",
```
Comment (romainmenke) [inline cssdb.settings.json:823]:
```suggestion
    "title": "`e`, `pi`, `infinity`, `-infinity` and `NaN` constants for calculation",
```

File: cssdb.settings.json [hunk 2]
```diff
@@ -809,6 +818,15 @@
...
+    "description": "Constants for calculation simplification",
+    "specification": "https://www.w3.org/TR/css-values-4/#calc-constants",
+    "stage": 2,
+    "browser_support": {},
+    "example": "p {\n  font-size: calc(calc(pow(e, pi) - pi) * 1rem);\n}"
```
Comment (romainmenke) [inline cssdb.settings.json:828]:
```suggestion
    "example": "p {\n  font-size: calc(pow(e, pi) * 1rem);\n}"
```

**Review by romainmenke — COMMENTED** [review]
Two small suggestions. Rest looks great :)

Maybe we can adjust the examples after we implement the plugins?
This will give us the chance to take a deep dive into the feature first.

File: cssdb.settings.json (hunk 1 above)
Comment (Antonio-Laguna) [reply cssdb.settings.json:823]: Nice catch!

**Pull request merged**